userguide/system.rst

```diff
@@ -2670,6 +2677,94 @@ Clicking |ui-options| for an entry shows these configuration buttons:
   request.
 
 
+.. _ACME Certificates:
+
+ACME Certificates
+~~~~~~~~~~~~~~~~~
+
+`Automatic Certificate Management Environment (ACME) <https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html>`__
+is available for automating certificate issuing and renewal. The user
+must verify ownership of the domain before certificate automation is
+allowed.
+
+ACME certificates can be created for existing certificate signing
+requests. These certificates use an :ref:`ACME DNS` authenticator to
+confirm domain ownership, then are automatically issued and renewed. To
+create a new ACME certificate, go to
+:menuselection:`System --> Certificates`,
+click |ui-options| for an existing certificate signing request, and
+click :guilabel:`Create ACME Certificate`.
+
+.. _ACME_cert_fig:
+
+.. figure:: images/system-acme-cert-add.png
+
+   ACME Certificate Options
+
+
+.. tabularcolumns:: |>{\RaggedRight}p{\dimexpr 0.22\linewidth-2\tabcolsep}
+                    |>{\RaggedRight}p{\dimexpr 0.15\linewidth-2\tabcolsep}
+                    |>{\RaggedRight}p{\dimexpr 0.62\linewidth-2\tabcolsep}|
+
+.. _ACME Certificate Options:
+
+.. table:: ACME Certificate Options
+   :class: longtable
+
+   +--------------------------------------+----------------+-------------------------------------------------------------------------------------+
+   | Setting                              | Value          | Description                                                                         |
+   +======================================+================+=====================================================================================+
+   | identifier                           | string         | Internal identifier of the certificate. Only alphanumeric, "_" and "-" are allowed. |
+   +--------------------------------------+----------------+-------------------------------------------------------------------------------------+
+   | Terms of Service                     | checkbox       | Please accept terms of service for the given ACME Server.                           |
+   +--------------------------------------+----------------+-------------------------------------------------------------------------------------+
+   | Renew certificate day                | integer        | Number of days to renew certificate before expiring                                 |
+   +--------------------------------------+----------------+-------------------------------------------------------------------------------------+
+   | ACME Server Directory URI            | drop-down menu | Please specify URI of ACME Server Directory.                                        |
+   +--------------------------------------+----------------+-------------------------------------------------------------------------------------+
+   | Authenticator for :samp:`{hostname}` | drop-down menu | Specify Authenticator to be used for :samp:`{hostname}`                             |
```

Comment:
"hostname" is not clear here--is it a field name or contents? Where is it found or entered? Also, "specify" is not needed. "to be used" is future tense.
```suggestion
   | Authenticator for :samp:`{hostname}` | drop-down menu | Authenticator to use for :samp:`{hostname}`                             |
```